rust/actyx/ax/src/cmd/users/devcert.rs

```diff
@@ -0,0 +1,48 @@
+use crate::{
+    certs::{AppDomain, DeveloperCertificateInput, ManifestDeveloperCertificate},
+    cmd::AxCliCommand,
+    crypto::{PrivateKey, PublicKey},
+    util::formats::{ActyxOSCode, ActyxOSResult, ActyxOSResultExt},
+};
+use futures::{stream::once, FutureExt, Stream};
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(version = crate::util::version::VERSION.as_str())]
+pub struct DevCertOpts {
```

Comment:
Note: This is moved from from https://github.com/Actyx/Actyx/blob/d52b140ca644114ca1ef3c2ba3c16f0d7c0764c4/rust/actyx/certs/src/bin/ax-cert.rs#L11